web/src/lib/components/shared-components/combobox.svelte

```diff
@@ -2,27 +2,37 @@
   // Necessary for eslint
   /* eslint-disable @typescript-eslint/no-explicit-any */
   type T = any;
+
+  export type Type = 'button' | 'submit' | 'reset';
```

Comment:
there is probably a provided type for this already. HtmlButtonElement['type'] or something